recipes/julia/build.sh

```diff
@@ -1,3 +1,3 @@
 #!/bin/sh
-make
-env PREFIX=$PREFIX prefix=$PREFIX make install
+mv * "$PREFIX"
+mv "$PREFIX/LICENSE.md" .
```

Comment:
It seems like the build process checks for the presence of the license file after running the `build.sh` or else it fails.